packages/auth-provider/auth-provider.ts

```diff
@@ -47,11 +45,11 @@ export class AuthProvider {
     }
 
     if (!param.apiGatewayUrl) {
-      param.apiGatewayUrl = API_GATEWAY_URL
+      param.apiGatewayUrl = EnvironmentUtils.fetchApiGwUrl()
     }
 
     if (!param.tokenEndpoint) {
-      param.tokenEndpoint = TOKEN_ENDPOINT
+      param.tokenEndpoint = EnvironmentUtils.fetchElementsAuthTokenUrl()
     }
```

Comment:
fail fast ?
is it supposed to work even if the their url and endpoint are `undefined/null` ?

Reply:
Not getting your point. This bit should not panic. If the dev doesn't provide the url we get it from the common package. The common package will always provide one based on the env, defaulting to prod.